src/System.IO/System.IO.sln

```diff
@@ -1,13 +1,13 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```

Comment:
Was this change done intentionally to see how many of the reviewers go through the whole PR? 😁 please revert, we are not changing the VS version before we migrate to runtime.

Reply:
@ViktorHofer Number of other .sln files in the repo have assorted selection of `16.*` versions. It is super annoying to keep fighting with VS changing these files.

Reply:
Should we gitignore sln files and require a force add when intentionally changing them?

Reply:
https://git-scm.com/docs/git-add#Documentation/git-add.txt--f

Reply:
also, is somebody on the VS side looking into making sln files more git friendly?

Reply:
> Should we gitignore sln files and require a force add when intentionally changing them?

Yes, I think it would be a fine idea.

Reply:
>  is somebody on the VS side looking into making sln files more git friendly?

I has been reported on VS feedback multiple times:

https://developercommunity.visualstudio.com/content/problem/181684/visual-studio-writes-version-in-solution-file.html
file.htmlhttps://developercommunity.visualstudio.com/content/problem/33838/visual-studio-preview-should-not-bump-sln-version.html

I do not know whether somebody is planning to actually do something about it.

Reply:
Thanks for the links. All of them are closed, which is really sad.